mpyg321/MPyg123Player.py

```diff
@@ -51,6 +52,15 @@ def unmute(self):
         """Unmutes the player"""
         self.player.sendline("UNMUTE")
 
+    def toggle_mute(self):
+        """Mute or UnMute if playing"""
+        if self.status == PlayerStatus.PLAYING:
```

Comment:
I believe you should remove this condition, because you can mute/unmute when you're not playing. See the test I did locally:
```
$ mpg123 --remote
@R MPG123 (ThOr) v10
MUTE
@mute
LOAD sample.mp3
@I {
@I }
@P 2
@S 1.0 3 48000 Joint-Stereo 2 768 2 0 0 0 256 0 1
@F 0 835 0.00 20.04
@F 0 835 0.00 20.04
@F 1 834 0.02 20.02
@F 2 833 0.05 19.99
@F 3 832 0.07 19.97
@F 4 831 0.10 19.94
@F 5 830 0.12 19.92
@F 6 829 0.14 19.90
@F 7 828 0.17 19.87
@F 8 827 0.19 19.85
```